boards/arm/mimxrt685_evk/init.c

```diff
@@ -42,32 +41,8 @@ static int mimxrt685_evk_pinmux_init(const struct device *dev)
 	SYSCTL1->FCCTRLSEL[3] |= SYSCTL1_FCCTRLSEL_DATAOUTSEL(1);
 #endif
 
-#endif
-
-#if DT_NODE_HAS_STATUS(DT_NODELABEL(red_pwm_led), okay)
```

Comment:
Not sure why this was missing in the initial conversion-thanks for catching it. Can you add `SCT0_OUT6_PIO0_31` to the `pinmux_sctimer` dts node, so this pinmux setting will still be applied?

Reply:
done